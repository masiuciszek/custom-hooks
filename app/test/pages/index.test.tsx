import React from "react";
import Home from "../../pages/index";
import { render } from "@testing-library/react";
import Layout from "../../components/layout";

describe("Home Page", () => {
  test("should ", () => {
    const home = render(
      <Layout>
        <Home />
      </Layout>
    );
    expect(home.asFragment()).toMatchSnapshot();
  });
});
