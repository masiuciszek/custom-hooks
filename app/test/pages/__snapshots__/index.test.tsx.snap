// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should Match Snapshot  1`] = `
<DocumentFragment>
  <nav
    class="navbar-sc-8gcs5t-0 dokJbT main-navbar"
  >
    <div
      class="Texts__SecondaryTitle-sc-5d7d2m-0 elCLXM"
    >
      <h4>
        React Hooks
      </h4>
    </div>
    <ul
      class="NavStyles__NavListStyles-q0dme3-0 kVznwr"
    >
      <li
        class="NavStyles__ListItem-q0dme3-1 bwqNHr"
        style="font-size: 1.8rem;"
      >
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li
        class="NavStyles__ListItem-q0dme3-1 bwqNHr"
        style="font-size: 1.8rem;"
      >
        <a
          href="/about"
        >
          About
        </a>
      </li>
    </ul>
    <ul
      class="NavStyles__NavListStyles-q0dme3-0 NavStyles__SmallListStyle-q0dme3-2 bkjBKr"
      style="transform: translateY(-1000%);"
    >
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/about"
        >
          About 
        </a>
      </li>
      <li>
        <a
          href="/faq"
        >
          Faq
        </a>
      </li>
    </ul>
    <span
      id="nav-icon"
    >
      üçî
    </span>
  </nav>
  <div
    class="layout__Page-sc-1lvij4u-0 kHeHsY"
  >
    <main
      class="layout__Main-sc-1lvij4u-1 hAlvDb"
    >
      <section
        class="title-sc-2igzia-0 cFwbxr Home-Title"
      >
        <h1>
          React Hooks
        </h1>
        <h3>
          <span>
            React
          </span>
           
          <span>
            hooks
          </span>
           
          <span>
            with
          </span>
           
          <span>
            Marcell
          </span>
        </h3>
      </section>
      <div>
        <h3>
          Choose theme
        </h3>
        <select
          id=""
          name=""
        >
          <option
            disabled=""
          >
            choose theme
          </option>
          <option
            selected=""
            value="light"
          >
            light
          </option>
          <option
            value="dark"
          >
            dark
          </option>
        </select>
      </div>
      <div
        class="Contentstyles__ContentStyles-sc-1q4sdj3-0 EMcTb"
      >
        <h2>
          A
        </h2>
        <h2>
          B
        </h2>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil cupiditate eligendi eos impedit nisi laboriosam saepe obcaecati officia, repellendus commodi assumenda quod excepturi voluptatem aliquam deserunt autem qui, consectetur quaerat.
        </p>
      </div>
    </main>
  </div>
  <footer
    class="footer-p4wu29-0 OtdOM MainFooter"
  >
    <div
      class="Texts__SecondaryTitle-sc-5d7d2m-0 elCLXM"
    >
      <h4>
        React Hooks
      </h4>
    </div>
    <div
      class="Footerstyles__FooterListWrapper-sc-1sn5qiw-0 hsnCZF"
    />
  </footer>
</DocumentFragment>
`;
